Segments/Oakvael.mapproj

```diff
@@ -118213,11 +118213,11 @@
           <block><![CDATA[]]></block>
         </script>
       </entry>
-      <entry weight="1">
+      <entry weight="4">
         <script name="OnCreate" enabled="true">
           <block><![CDATA[]]></block>
           <block><![CDATA[
-	return randomBalm(true);
```

Comment:
Wouldn't you want to replace stamina pots with this? 

ItemEntity OnCreate(MobileEntity from, Container container)
{
	return new StaminaPotion();
}

I think Lux would want this for randomization of bad bottles. (orc/dragon/troll urine)

![image](https://user-images.githubusercontent.com/90510109/210243782-fb5eaf14-5250-41cc-9b2f-539c570aebfc.png)

Reply:
![image](https://user-images.githubusercontent.com/90510109/210243845-fb403911-984a-42f9-91c4-412eb35994ad.png)

Reply:
iirc, some random stam pots dropped alongside falcon stam pots.

The double entry on randombalms seemed a strange redundancy, and at 2.7% chance - seemed ideal for replacement.

Reply:
ohhh.. thats the urine!  nm..   will pull it back.